src/features/styles/theme/table.scss

```diff
@@ -32,6 +32,6 @@
 }
 
 th, td {
-    line-height: 1.5;
+    line-height: 1.8!important;
```

Comment:
why !important?

Reply:
Hi @hasan-deriv Without important it is not picking line height .
You can suggest other way. I can try tha.